scripts/spack/specs.json

```diff
@@ -14,9 +14,7 @@
     "__comment__":"##############################################################################",
 
     "toss_4_x86_64_ib":
-    [ "gcc@10.3.1+devtools+profiling",
-      "clang@14.0.6+devtools+profiling",
-      "clang@16.0.6+devtools+profiling~openmp" ],
+    [ "clang@19.1.3+devtools+profiling+enzyme" ],
```

Comment:
so beyond this point is serac no longer going to support gcc builds due to it being incompatible with enzyme?

Reply:
just so i can communicate things to the lido team properly, we're removing the gcc build because serac is going all-in on enzyme support, which is incompatible with gcc. so next time we update lido, its gcc builds will have to be mfem-only or removed. is that right?